book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './rental'
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
```

Comment:
- [ ] @ger619 the rental should have access to read and write like the title and author, kindly move it to attr_accessor.